core/chains/evm/headtracker/head_broadcaster_test.go

```diff
@@ -21,6 +22,20 @@ import (
 	"github.com/smartcontractkit/chainlink/core/utils"
 )
 
+func waitHeadBroadcasterToStart(t *testing.T, hb types.HeadBroadcaster) {
+	t.Helper()
+
+	subscriber := &cltest.MockHeadTrackable{}
+	_, unsubscribe := hb.Subscribe(subscriber)
+	defer unsubscribe()
+
+	hb.BroadcastNewLongestChain(cltest.Head(1))
+	g := gomega.NewWithT(t)
+	g.Eventually(func() int32 {
+		return subscriber.OnNewLongestChainCount()
+	}).Should(gomega.Equal(int32(1)))
```

Comment:
Does the signature match?
```suggestion
	g.Eventually(subscriber.OnNewLongestChainCount).Should(gomega.Equal(int32(1)))
```